public/rulesets/default/2_3_2_Content_Type

```diff
@@ -0,0 +1,24 @@
+rules:  
+  - id: "2.3.2"
+    title: "Content-Type: Default Media Type"
+    message: "If the request or response body is present, the default media type must be application/json with charset=UTF-8 or application/bson."
+    option: Mandatory
+    location: path
+    element:
+  	- requestBody
+  	- responses
+    call:
+  	function: mediaTypeCheck
+  	functionParams:
+  	  headers:
+  		- "application/json; charset=UTF-8"
+  		- "application/json; charset=utf-8"
+  		- "application/json; charset=\"utf-8\""
+  		- "application/octet-stream"
+  		- "multipart/form-data"
+  		- "application/bson"
+  	  methods:
+  		- "post"
+  		- "put"
+  		- "delete"
+    severity: "high"
```

Comment:
add one more rule under the id 2.3.2.1 which will address situation that content-type header must not be defined for GET requests in parameters section